content/docs/rfcs/17/README.md

```diff
@@ -203,6 +212,13 @@ With a reasonable approximation of the preceding values, one can set  `max_epoch
 `Network_Delay` and `Clock_Asynchrony` MUST have the same resolution as  `period` .
 By this formulation,  `max_epoch_gap`  indeed measures the maximum number of `epoch`s that can elapse since a message gets routed from its origin to all the other peers in the network.
 
+`acceptable_root_window_size` depends upon the underlying chain's average blocktime, and `Network_Delay`
+
+`acceptable_root_window_size` SHOULD be set as $acceptable_root_window_size=block_time/Network_Delay$
```

Comment:
Can you please elaborate on the reasoning behind this formula?  To me, it should be the other way around i.e., Net delay/ block time
if a messages takes X ms to reach the entire network, then we should measure that within that X span, how many blocks may get mined (due to which the tree state may get updated), then that number of mined blocks during X would be an approximation of the window size (it is a lower bound but not upper bound).
We also need to account for the block propagation delay in the Ethereum network as well, i.e.,  the time that it takes until all the Ethereum clients become aware of the new block.

Reply:
Completely right! I didn't think of it that way. Will address soon.

Reply:
Addressed in [dc3d476](https://github.com/vacp2p/rfc/pull/539/commits/dc3d476848c3ca73c3fa9c45fdca020b82d84a22)

Reply:
Thanks for addressing it, just curious to know why `block_propagation_delay` is in the denominator, can you please add further reasoning about this into the RFC? it will make it easier for the readers to relate to the formulation as well.

Reply:
Initially I had thought that we were measuring the acceptable window of roots by number of blocks generated during the network delay. However, the network delay is very small compared to the block time of any blockchain,  and therefore the number of acceptable window of roots should be calculated based on the formula proposed by you.

Reply:
Addressed in b76f6be

Reply:
For ease of explanation, let's assume `acceptable_root_window_size= block_propagation_delay` is already part of the `block_time`, simplifying the formula to `Network_delay/block_time`.